internal/stacks/stackruntime/apply_test.go

```diff
@@ -3110,16 +3117,14 @@ func TestApplyWithChangedInputValues(t *testing.T) {
 
 	go Apply(ctx, &applyReq, &applyResp)
 	applyChanges, applyDiags := collectApplyOutput(applyChangesCh, diagsCh)
-	if len(applyDiags) != 1 {
```

Comment:
this was redundant, we also check the diagnostics on line 3121, so doing it twice was pointless.